packages/react-components/react-teaching-popover-preview/src/components/TeachingPopoverBody/useTeachingPopoverBody.ts

```diff
@@ -1,34 +1,41 @@
 import * as React from 'react';
-import { getIntrinsicElementProps, slot } from '@fluentui/react-utilities';
+import { getIntrinsicElementProps, resolveShorthand } from '@fluentui/react-utilities';
 import type { TeachingPopoverBodyProps, TeachingPopoverBodyState } from './TeachingPopoverBody.types';
-
 /**
- * Create the state required to render TeachingPopoverBody.
- *
- * The returned state can be modified with hooks such as useTeachingPopoverBodyStyles_unstable,
- * before being passed to renderTeachingPopoverBody_unstable.
- *
- * @param props - props from this instance of TeachingPopoverBody
- * @param ref - reference to root HTMLDivElement of TeachingPopoverBody
+ * Returns the props and state required to render the component
+ * @param props - TeachingPopoverBody properties
+ * @param ref - reference to root HTMLElement of TeachingPopoverBody
  */
 export const useTeachingPopoverBody_unstable = (
   props: TeachingPopoverBodyProps,
-  ref: React.Ref<HTMLDivElement>,
+  ref: React.Ref<HTMLElement>,
 ): TeachingPopoverBodyState => {
+  const { as, media, mediaLength } = props;
+
+  let mediaShorthand;
+
+  // Image shouldn't be rendered if its src is not set
+  if (media?.src) {
+    mediaShorthand = resolveShorthand(media, {
+      required: false,
+      defaultProps: {
+        alt: '',
+        role: 'presentation',
+        'aria-hidden': true,
+      },
+    });
+  }
+
   return {
-    // TODO add appropriate props/defaults
     components: {
-      // TODO add each slot's element type or component
       root: 'div',
+      media: 'img',
```

Comment:
This API is not complaint with other components in Fluent as `media` renders `img`. I suggest to do following:
- make `media` slot `span`
- users will be able to pass any content there, for example, `<TeachingPopoverBody media={<Image src="" } />`